pkgs/tools/networking/xdp-tools/default.nix

```
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libbpf
+, elfutils
+, libelf
+, zlib
+, libpcap
+, clang
+, llvm
+, gnumake
+, gcc
+, pkgconfig
+, m4
+, emacs-nox
+, wireshark-cli
+}:
+stdenv.mkDerivation rec {
+  pname = "xdp-tools";
+  version = "1.2.5";
+
+  src = fetchFromGitHub {
+    owner = "xdp-project";
+    repo = "xdp-tools";
+    rev = "v${version}";
+    sha256 = "sha256-Kyay5j+87nOZ9C+DI8MI6zhkWqspIHiTfW9Di5uVWzY=";
+  };
+
+  buildInputs = [
+    libbpf
+    elfutils
+    libelf
+    libpcap
+    zlib
+  ];
+
+  nativeBuildInputs = [
+    clang
+    llvm
+    gnumake
+    gcc
+    pkgconfig
+    m4
+    emacs-nox
+    wireshark-cli
+  ];
+
+  BPF_CFLAGS = "-fno-stack-protector -Wno-error=unused-command-line-argument";
+  PRODUCTION = 1;
+  DYNAMIC_LIBXDP = 1;
+  FORCE_SYSTEM_LIBBPF = 1;
+  FORCE_EMACS = 1;
+
+  installPhase = ''
+    export PREFIX=$out
+
+    make install
+  '';
```

```suggestion
  makeFlags = [ "PREFIX=$(out)" ];
```
This can be simpler.

---

Done.